pkg/config/config_writer.go

```diff
@@ -308,6 +308,14 @@ func extractConfigType(context *serializerContext, config Config) (typeDefinitio
 				Resource: t.Resource,
 			},
 		}, nil
+	case BucketType:
```

Comment:
Do we even want to write anything yet? IMO by the time we have bucket downloads the YAML format should be specific and not reuse the API definition. 

I'd just drop this case and let any writes fall into the default error case until we actually handle downloading and writing

Reply:
Dropped